src/components/Radio/RadioItem/RadioItem.module.css

```diff
@@ -0,0 +1,102 @@
+.radioInput {
+  height: 0;
+  width: 0;
+  opacity: 0;
+  position: absolute;
+}
+
+.radioInput + .radioButtonContainer {
+  cursor: pointer;
+  display: flex;
+  border-radius: var(--border-radius-small);
+  padding: 10px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex: 1 1 0;
+  width: auto;
+}
+
+.radioInput + .radioButtonContainer .header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.radioInput + .radioButtonContainer .header .checkBox {
+  width: 16px;
+  height: 16px;
+  border-radius: 50%;
+  border: solid 2px var(--font-color-sub);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 14px;
+}
+
+.radioInput + .radioButtonContainer .header .checkBox .checkBoxFilling {
+  width: 0px;
+  height: 0px;
+  border-radius: 50%;
+  background-color: var(--font-color-sub);
```

Comment:
I think this was by mistake. A font color is used as background color here.

Reply:
Good catch